backend/users/serializers.py

```
@@ -27,6 +27,48 @@ def _is_inactive_test():
+            )
+
+        temp_user = User(
+            email=attrs["email"],
+            first_name=attrs["first_name"],
+            last_name=attrs["last_name"],
+        )
+
+        try:
+            validate_password(password=password, user=temp_user)
+        except DjangoValidationError as e:
+            raise serializers.ValidationError({"password": list(e.messages)})
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)